app/code/community/Reverb/ProcessQueue/Model/Source/Task/Status.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Author: Sean Dunagan
+ * Created: 9/14/15
+ */
+
+class Reverb_ProcessQueue_Model_Source_Task_Status
+{
+    protected $_options = null;
+
+    public function getAllOptions()
+    {
+        if (is_null($this->_options)) {
+            $this->_options = array(
+                array(
+                    'label' => Mage::helper('reverb_process_queue')->__('Pending'),
+                    'value' => Reverb_ProcessQueue_Model_Task::STATUS_PENDING
+                ),
+                array(
+                    'label' => Mage::helper('reverb_process_queue')->__('Processing'),
+                    'value' => Reverb_ProcessQueue_Model_Task::STATUS_PROCESSING
+                ),
+                array(
+                    'label' => Mage::helper('reverb_process_queue')->__('Complete'),
+                    'value' => Reverb_ProcessQueue_Model_Task::STATUS_COMPLETE
+                ),
+                array(
+                    'label' => Mage::helper('reverb_process_queue')->__('Error'),
+                    'value' => Reverb_ProcessQueue_Model_Task::STATUS_ERROR
+                ),
+                array(
+                    'label' => Mage::helper('reverb_process_queue')->__('Aborted'),
+                    'value' => Reverb_ProcessQueue_Model_Task::STATUS_ABORTED
```

Comment:
Aborted means it should not be re-attempted e.g. an order was already synced from Reverb